.github/workflows/codeql-analysis.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ master, v10, v11 ]
+    branches: [ master, v11, v12, prepare-v14 ]
```

Comment:
- [ ] create branch `v12` before merging
- [ ] remove this branch from CI before merging